usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+
+message distset
+{A distribui�ao NetBSD � dividida em uma cole�ao de conjuntos de distribui�ao.
+H� alguns conjuntos b�sicos que sao necess�rios para todas as instala�oes
+e h� alguns outros conjuntos que sao opcionais. Voc� pode escolher instalar
+um conjunto principal (instala�ao m�nima), todos eles (instala�ao completa) ou um
+grupo personalizado de conjuntos (instala�ao personalizada).
+}
+
+/* Called with: 			Example
+ *  $0 = sets suffix			.tgz
+ *  $1 = URL protocol used		ftp
+ */
+message ftpsource
+{A seguir estao o site $1, diret�rio, usu�rio e senha que
+serao usados. Se "usu�rio" for "ftp", entao a senha nao � necess�ria.
+
+}
+
+message email
+{endere�o de email}
+
+message dev
+{dispositivo}
+
+/* Called with: 			Example
+ *  $0 = sets suffix			.tgz
+ */
+message nfssource
+{Insira o diret�rio do host e servidor nfs onde a distribui�ao est� localizada.
+Lembre-se, o diret�rio deve conter os arquivos $0 e deve ser mont�vel em nfs.
+
+}
+
+message floppysource
+{Insira o dispositivo de disquete a ser usado e o diret�rio de transfer�ncia no sistema de arquivos
+alvo. Os arquivos definidos devem estar no diret�rio raiz dos disquetes.
+
+}
+
+/* Called with: 			Example
+ *  $0 = sets suffix			.tgz
+ */
+message cdromsource
+{Insira o dispositivo de CDROM a ser usado e o diret�rio no CDROM onde
+a distribui�ao est� localizada.
+Lembre-se, o diret�rio deve conter os arquivos $0.
+
+}
+
+message No_cd_found
+{Nao foi poss�vel localizar uma m�dia de CD em nenhuma unidade com os conjuntos de distribui�ao!
+Insira os dados corretos manualmente ou insira um novo disco e tente novamente.
+}
+
+message abort_install
+{Cancelar instala�ao}
+
+message source_sel_retry
+{Voltar para a sele�ao da fonte & tentar novamente}
+
+message Available_cds
+{CDs dispon�veis }
+
+message ask_cd
+{V�rios CDs encontrados. Selecione o que cont�m o CD de instala�ao.}
+
+message cd_path_not_found
+{Os conjuntos de instala�ao nao foram encontrados no local padrao neste
+CD. Verifique o dispositivo e o nome do caminho.}
+
+/* Called with: 			Example
+ *  $0 = sets suffix			.tgz
+ */
+message localfssource
+{Insira o dispositivo local desmontado e o diret�rio naquele dispositivo onde
+a distribui�ao est� localizada.
+Lembre-se, o diret�rio deve conter os arquivos $0.
+
+}
+
+/* Called with: 			Example
+ *  $0 = sets suffix			.tgz
+ */
+message localdir
+{Entre no diret�rio local j� montado onde a distribui�ao est� localizada.
+Lembre-se, o diret�rio deve conter os arquivos $0.
+
+}
+
+message filesys
+{sistema de arquivo}
+
+message nonet
+{Nao consigo encontrar nenhuma interface de rede para uso pelo NetBSD. Voc� ser�
+retornado ao menu anterior.
+}
+
+message netup
+{As seguintes interfaces de rede estao ativas: %s
+Alguma delas se conecta ao servidor necess�rio?}
+
+message asknetdev
+{Qual dispositivo de rede voc� gostaria de usar?}
+
+message netdevs
+{Interfaces dispon�veis}
+
+message netinfo
+{To be able to use the network, we need answers to the following:
```

Comment:
You might have forgotten to translate this.
My suggestion: "A fim de poder utiliar a rede, são necessárias respostas para as seguintes questões:"